pkg/auth/auth.go

```diff
@@ -15,5 +15,7 @@ type Config struct {
 }
 
 func Start(ctx context.Context, config Config, authZ auth.Config) error {
-	return errors.ThrowUnimplemented(nil, "AUTH-l7Hdx", "not implemented yet") //TODO: implement
+	err := api.Start(ctx, config.API)
+	logging.Log("MAIN-udjed").OnError(err).Panic("unable to start api")
+	return err
```

Comment:
panic... no return needed

Reply:
I return the error and remove panic, because in main we have a fatal